docs/content/en/_index.md

```diff
@@ -12,44 +12,25 @@ description: "K8ssandra documentation: architecture, configuration, guided tasks
 type: docs
 ---
 
-The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. Check out the sections at left based on your needs and be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
+The K8ssandra documentation covers everything from install details, deployed components, configuration references, and guided outcome-based tasks. 
 
-{{% alert title="Tip" color="primary" %}}
-If you're impatient, jump right in with our **[install]({{< relref "install" >}})** topics!
-{{% /alert %}}
+Be sure to leave us a <a class="github-button" href="https://github.com/k8ssandra/k8ssandra" data-icon="octicon-star" aria-label="Star k8ssandra/k8ssandra on GitHub">star</a> on Github!
 
-## What is K8ssandra?
+## Features for single- and multi-cluster Kubernetes environments
```

Comment:
So I think this table content is good, but where do we want this to live?

Would this go on the "common" docs set? If so this stuff should go in a separate PR targeting the `docs` branch.  Or do we want this on both versions -- basically everywhere?  I would have though it should go in the common set.

Reply:
Yes, after this commit, I will do surgery to get this into the common (docs branch) content.